<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelDialog</class>
 <widget class="QDialog" name="modelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Parameter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Value</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="paramLabel_3">
        <property name="text">
         <string>Param 3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="paramLineEdit_3"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="paramLineEdit_1"/>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="paramCheckBox_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="paramLabel_4">
        <property name="text">
         <string>Param 4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="paramLabel_1">
        <property name="text">
         <string>Param 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="paramLineEdit_2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="paramLabel_2">
        <property name="text">
         <string>Param 2</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="paramLabel_7">
        <property name="text">
         <string>Param 7</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="paramLineEdit_5"/>
      </item>
      <item row="7" column="2">
       <widget class="QCheckBox" name="paramCheckBox_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="paramLineEdit_7"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="paramLabel_6">
        <property name="text">
         <string>Param 6</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="paramLabel_9">
        <property name="text">
         <string>Param 9</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="paramLabel_5">
        <property name="text">
         <string>Param 5</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="paramLabel_8">
        <property name="text">
         <string>Param 8</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="paramLineEdit_9"/>
      </item>
      <item row="10" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="paramLabel_10">
        <property name="text">
         <string>Param 10</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="paramLineEdit_4"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="paramLineEdit_8"/>
      </item>
      <item row="11" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Fixed</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2" colspan="2">
       <widget class="QCheckBox" name="paramCheckBox_7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="paramLineEdit_10"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="paramLineEdit_6"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="modelName">
        <property name="text">
         <string>model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
